batch/postprocessing-scripts.sh

```diff
@@ -1,7 +1,13 @@
 
 # START: your postprocessing scripts goes here.
 
+export FULL_FIT=TRUE
+export DIAGNOSTICS=TRUE
+Rscript $FLEPI_PATH/postprocessing/run_sim_processing_SLURM.R -c $CONFIG_PATH 
 
+export FULL_FIT=FALSE
```

Comment:
l.8-10 were redundant (already seen in l.4-6) so remove these

Reply:
It sets up the script run_slim_processing in two different fashion. One plotting the full fits and one with just the forecasts